freshmaker/views.py

```diff
@@ -744,6 +752,20 @@ def get(self, project, repo):
         return jsonify(ret), 200
 
 
+class PullsecOverrideAPI(MethodView):
+    @freshmaker_build_api_latency.time()
+    def get(self, id):
+        if id is None:
```

Comment:
This should not be possible since the API route in Flask will not match if there is no ID provided.

Reply:
Yes, it works. But it doesn't show pullsec_overritdes in the common `builds` endpoint. And it returns just `null` if there are no pullspecs overrides. But since it wasn't mentioned in the tickets I didn't do anything with it.